File: Laboratorio02/CodigoJava/Reporte.java
public class Reporte {

    public void generarReporte(Estudiante estudiante) {
        System.out.println("REPORTE DEL ESTUDIANTE");
        System.out.println("Nombre: " + estudiante.getNombre());
        System.out.println("Edad: " + estudiante.getEdad());
        System.out.println();
    }

    @Override
    public String toString() {
        return "Reporte{}";
    }
}

File: Laboratorio02/CodigoPython/reporte.py
class Reporte:
    def generar_reporte(self, estudiante):
        print("REPORTE DEL ESTUDIANTE")
        print(f"Nombre: {estudiante.nombre}")
        print(f"Edad: {estudiante.edad}")
        print()

    def __str__(self):
        return "Reporte{}"
